node/service/src/rpc.rs

```diff
@@ -97,6 +95,10 @@ pub struct FullDeps<C, P, A: ChainApi, BE> {
 	pub fee_history_cache: FeeHistoryCache,
 	/// Channels for manual xcm messages (downward, hrmp)
 	pub xcm_senders: Option<(flume::Sender<Vec<u8>>, flume::Sender<(ParaId, Vec<u8>)>)>,
+	/// Ethereum data access overrides.
+	pub overrides: Arc<OverrideHandle<Block>>,
```

Comment:
What is this "overrides" ?

Reply:
Moved from parameters to membre of `FullDeps` struct.
I tried to stay similar to the frontier template for that part.